src/main/java/API/UserParser.java

```diff
@@ -1,20 +1,14 @@
 package API;
 
 import Controllers.UserController;
-import Entities.User;
+import Entities.*;
```

Comment:
Just a quick question. If I remember correctly, UserParser is in the Controller layer. If that's the case, then shouldn't we completely remove the import statement for entities? Since we are still keeping it there wouldn't that violate the clean architecture?